features/192-asset-gear/index.md

```diff
@@ -70,6 +71,7 @@ This endpoint requires API Key and authenticated user.
 }
 ```
 
+- `request_kind`: The kind of the presigned request. `identity` is the default. If it is `identity`, the data must be the request body. If it is `multipart/form-data`, the data must be encoded in `multipart/form-data`.
```

Comment:
The term `identity` comes from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding

Reply:
The `content-type` must be the actual content type because the implementation is going to transform the multipart form request into a PUT Object request. We still need the actual content type so that the content type is correct.

Reply:
The multipart form data should contains the actual content type, so we can delay initiating the proxied request until the content type is known.

ref: https://stackoverflow.com/a/23517227

```
-----------------------------9051914041544843365972754266
Content-Disposition: form-data; name="file1"; filename="a.txt"
Content-Type: text/plain

Content of a.txt.
```